modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/ItIgniteInMemoryNodeRestartTest.java

```diff
@@ -200,15 +200,9 @@ public void inMemoryNodeRestartNotLeader(TestInfo testInfo) throws Exception {
         InternalTableImpl internalTable = (InternalTableImpl) restartingTable.internalTable();
 
         // Check that it restarts.
-        waitForCondition(
+        assertTrue(waitForCondition(
                 () -> isRaftNodeStarted(table, loza) && solePartitionAssignmentsContain(restartingNodeConsistentId, internalTable),
-                TimeUnit.SECONDS.toMillis(10)
-        );
-
-        assertTrue(isRaftNodeStarted(table, loza), "Raft node of the partition is not started on " + restartingNodeConsistentId);
```

Comment:
Because above we checks both of lower assertions together. If await assertion fails => we couldn't move here. If it passes => both assertions are true and excessive there.